packages/backend/src/models/OpenIdIdentitiesModel.ts

```diff
@@ -53,6 +53,14 @@ export class OpenIdIdentityModel {
             );
     }
 
+    async findIdentitiesByEmail(email: string): Promise<OpenIdIdentity[]> {
```

Comment:
this returns all OpenId's with an openid email matching the query email and doesn't account for primary emails on Lightdash user accounts ✅